src/js/recommendation-ui.js

```
@@ -120,15 +120,56 @@ document.addEventListener('DOMContentLoaded', () => {
+      const fileName = file.name.toLowerCase();
+
+      if (fileName.endsWith('.pdf')) {
+        try {
+          if (typeof pdfjsLib === 'undefined') {
+            showError("PDF library not loaded. Please try again after refreshing the page.");
+            e.target.value = '';
+            return;
+          }
+          resumeText.placeholder = "Parsing PDF...";
+          const arrayBuffer = await file.arrayBuffer();
+          const pdf = await pdfjsLib.getDocument({ data: arrayBuffer }).promise;
```

ESLint

[error] 'pdfjsLib' is not defined.

(no-undef)